.github/workflows/maven-publish.yml

```diff
@@ -44,14 +44,13 @@ jobs:
           java-version: '11'
           distribution: 'temurin'
           cache: 'maven'
-          server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
```

Comment:
In the GitHub documentation example, the `server-id` line is kept:
https://docs.github.com/en/actions/publishing-packages/publishing-java-packages-with-maven#publishing-packages-to-the-maven-central-repository-and-github-packages

update: seems the default value is "github", so you want to keep the line:
https://github.com/actions/setup-java#maven-options

Reply:
In the GH doc above, they provide some corresponding content of the generated settings.xml - you should "cat" the one(s) generated in your workflow and compare, making sure the parameters you pass tp `setup-java` are reflected in there.

Reply:
`ossrh` is for maven central AFAIK
`github` is for maven central